R/serializers.R

```diff
@@ -1,3 +1,21 @@
+#' Add a function for serializing an input before bookmarking application state
+#'
+#' @param inputId Name of the input value.
+#' @param fun A function that takes the input value and returns a modified
+#'   value. The returned value will be used for the test snapshot.
+#' @param session A Shiny session object.
+#'
+#' @export
+serializer <- function(inputId, fun, session = getDefaultReactiveDomain()) {
```

Comment:
How would you feel about `session$setSerializer(inputId, fun)` instead? Or at least renaming to `setSerializer`? This looked more to me like a constructor at first, something that makes a serializer out of a function; rather than the side-effecty, accessor-y operation that it actually is.